.github/workflows/python-package.yml

```diff
@@ -55,3 +55,13 @@ jobs:
       with:
         name: report.html
         path: report.html
+
+    - name: Coverage Report
```

Comment:
In the future, both steps that use pytest should be unified in single one 
Anyway it is taking less than 10 seconds, so it is ok for now